linker/BUILD.bazel

```diff
@@ -0,0 +1,11 @@
+load("//js:nodejs_package.bzl", "nodejs_package")
+
+nodejs_package(
+    name = "linker",
+    package_name = "@rules_js/linker",
```

Comment:
what does `@rules_js` mean here? we don't have such a scope on npm and don't plan to publish one. do we actually resolve from this package name?

Reply:
Kind of yes. the reason I gave it the name is because rule authors will want to invoke it at a given time so it has to be reachable to the users. One might add a dependency on it like `@aspect_rules_js//linker` since it is released with the rules itself rather than on npm.  One use case that comes to my mind is a rule that has worker mode enabled thus has to invoke this to link again.

Honestly, this linker package is just a tiny function that links from `MERGED_NODE_MODULES`  to `pwd/node_modules`. Basically `ln -s $MERGED_NODE_MODULES $(pwd)/node_modules`.  I am not sure if we should have this at all.

Reply:
I was thinking maybe we should get rid of NODE_PATH and change the laucher script to link from MERGED_NODE_MODULES which would point to `runfiles/node_modules` for statically linked modules and `bazel-out/node_modules` for dynamically linked modules (merged via merge_deps.bzl hence it has all needed dependencies)

so in both cases, we just rely on bazel to create the node_modules tree and then we symlink that directory next to CWD.